<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="sinTab">
       <attribute name="title">
        <string>sin(7x)</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="cosTab">
       <attribute name="title">
        <string>cos(2x)</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="convTab">
       <attribute name="title">
        <string>Свертка</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="corrTab">
       <attribute name="title">
        <string>Корреляция</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="furConvTab">
       <attribute name="title">
        <string>Свертка Фурье</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="furCorrTab">
       <attribute name="title">
        <string>Корреляция Фурье</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
